src/config/data/faq.js

```diff
@@ -4,151 +4,98 @@ const liSpace = { marginBottom: "10px" };
+        there’s a place for you at ShellHacks!
       </>
     )
   },
   {
-    title: <>Do I need to already have a group?</>,
+    title: <>Do I need to have a group?</>,
     description: (
       <>
-        Nope. You can go solo, come with your own team (no more than 4), or
-        participate in a group mixer we'll have before hacking begins to find a
-        group.
+        Not at all! You can be a lone wolf, come with a premade team (no more than four people), 
+        or even join some teams at the event. We’ll also have team building activities to help you find the right teammates!
       </>
     )
   },
   {
-    title: <>Will there be travel reimbersement?</>,
+    title: <>How much does it cost?</>,
     description: (
       <>
-        We're currently working those details out! Come back later or checkout "Essentials" once details are finalized.
-        {/* There will be travel reimbursement available with the following
-        stipulations:
-        <br />
-        <br />
-        <ol style={{ listStylePosition: "inside", listStyleType: "decimal" }}>
-          <li style={liSpace}> Up to $60 in gas dependent on mileage</li>
-          <li style={liSpace}>
-            Must not be coming from a school which has a bus (unless the bus is
-            filled)
-          </li>
-          <li style={liSpace}>
-            All cars must be filled with a minimum of 4 passengers
-          </li>
-          <li style={liSpace}>
-            All passengers must be attending the hackathon
-          </li>
-        </ol>
-        We will be sending information on how to claim the reimbursement at a
-        later date, so{" "}
-        <span style={{ fontWeight: "bold" }}>
-          please keep hold of your receipts!
-        </span> */}
+        Nothing! That’s right, ShellHacks is entirely free for 
+        all attendees to participate. All you need to worry about is learning new skills, developing 
+        cool projects, and having fun!
       </>
     )
   },
-
   {
-    title: <>Alright, I'm hyped for ShellHacks! How do I register?</>,
+    title: <>Will food be provided?</>,
     description: (
       <>
-        Just click the big "Register" button up top. Make sure you check your
-        email too since we'll be having your verify your email address and
-        sending acceptances through there too!
+        Yes! All major meals and snacks will be provided to everyone attending ShellHacks. We'll also have coffee, energy drinks, and even tea, for those 
+        sophisticated hackers out there!
+        <br />
+        <br />
+        We’ll do our best to accommodate any dietary restrictions as well; just note them on your 
+        registration form and we’ll make sure you get what you need.
       </>
     )
   },
   {
-    title: <>So, why ‘ShellHacks’?</>,
+    title: <>Will there be travel reimbersement?</>,
     description: (
       <>
-        Well, it started off as a joke about what a turtle-themed hackathon

```

Comment:
what was wrong with this?

Reply:
Theres no need to have that FAQ anymore. They were getting long so I removed this